nilearn/tests/test_numpy_conversions.py

```diff
@@ -230,3 +231,15 @@ def test_as_ndarray():
     ## Unhandled cases
     assert_raises(ValueError, as_ndarray, "test string")
     assert_raises(ValueError, as_ndarray, [], order="invalid")
+
+
+def test_csv_to_array():
+    # Create a phony CSV file
+    with tempfile.NamedTemporaryFile(suffix='.csv', delete=False,
+                                     mode='wt') as csv_file:
+        csv_file.write('1.,2.,3.,4.,5.\n')
```

Comment:
BTW just curious, why delete=False and os.remove at the end ? I would have thought that the temporary file would be deleted just fine at the exit of the context generator.